applications/vanilla/views/categories/helper_functions.php

```diff
@@ -93,9 +93,15 @@ function getOptions($category) {
 endif;
 
 if (!function_exists('MostRecentString')):
-    function mostRecentString($row) {
-        if (!$row['LastTitle'])
+    function mostRecentString($row, $options = []) {
+        $options = (array)$options + [
+            'user' => true,
+            'date' => true,
```

Comment:
These aren't really named in a way that your would traditionally name a boolean option.

I would expect `user` to be an actual user record, and `date` to be a date object.

`displayUser` & `displayDate` would be more appropriate (or something else).